crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -2856,6 +3050,63 @@ Ztt..Kttttt..ttZ
 ZtrrrrtZZtqqqqtZ
 ENDMAP
 
+NAME:    nicolae_swamp_thats_no_hydra
+TAGS:    transparent no_pool_fixup
+DEPTH:   Swamp
+KMONS:   1 = kraken
```

Comment:
I think this one needs something a bit "extra" for the theme to be worthwhile. Not sure what, my first thought is hydra skeletons.